zigpy_deconz/zigbee/application.py

```diff
@@ -281,8 +288,13 @@ def handle_tx_confirm(self, req_id, status):
             )
 
 
-class ConBeeDevice(zigpy.device.Device):
+class DeconzDevice(zigpy.device.Device):
     """Zigpy Device representing Coordinator."""
+    def __init__(self, version: int, device_path: str, *args):
+        super().__init__(*args)
+        is_serial_devcie = re.match(r"/dev/tty(S|AMA)\d+", device_path)
```

Comment:
Should we handle `/dev/serial/by-id/` cases?

Reply:
My current understanding is, that `/dev/serial/by-id/` is used for USB devices. So whatever show up under that is a USB device and therefore a ConBee. Can GPIO shields like the RaspBee also show up under `/dev/serial/by-id/`?

Maybe the name ism't the best, as USB devices are also serial devices, duh. I should probably change that to `is_shield_device` or  `is_gpio_device`.

Reply:
I think you are right.